core/src/main/scala/akka/kafka/internal/KafkaConsumerActor.scala

```diff
@@ -608,14 +603,19 @@ import scala.util.control.NonFatal
               commitRefreshing.committed(offsets)
               replyTo.foreach(_ ! Done)
 
+            case e: RebalanceInProgressException =>
+              log.info("Kafka commit is to be retried, after={} ms, commitsInProgress={}, cause={}",
```

Comment:
Include the logging in the method too to reduce duplication.

Reply:
I thought that we don't need "warning" level on such retries, but I don't mind